src/main/java/org/randywebb/wlts/reports/DetailedMemberListCSV.java

```diff
@@ -36,15 +40,22 @@
         @throws IOException on network errors
         @throws ParseException If the JSON was not the format that was expected
     */
-    public static void generateWLTSReport(LdsToolsClient client, String filePath) throws IOException, ParseException {
+    public static void generateWLTSReport(LdsToolsClient client, String filePath)
+                                                throws IOException, ParseException {
         // Parse JSON Membership file into beans
 
         InputStream in = client.getMemberInfo();
 
         List<DetailedMember> members = processDetailMembers(in);
 
-        // List<DetailedMember> members = processDetailMembers(Thread.currentThread().getContextClassLoader().getResourceAsStream("detailedmembership.json"));
-        // List<Household> households = processHouseholds(Thread.currentThread().getContextClassLoader().getResourceAsStream("membership.json"));
+        /*
```

Comment:
Ugh, I really don't like this automated wrap. This isn't a commented block, but a couple of troubleshooting lines of code that I enable to run against a test file. Now it is harder to switch them on and off.

Reply:
Ah, ok, now that I know that I can clean this up to make them on/off-able

Reply:
It's not that big of a deal, sorry for the rant (I was tired). This was an example of why I don't like hard-enforced line length limits. It looked like an auto formatter had wrapped all the lines to a very short length.

Really, I shouldn't have that kind of quick switch in the comments. That's not clean code, but it was convenient for remembering how to do a quick test.
I'll eventually move that to the unit tests instead of a commented line that I can turn on and off.